src/test/java/unit/Group.java

```diff
@@ -315,6 +294,32 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private void expectListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // call `terra group list-users` in table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
+
+    // assert header is correct
+    assertTrue(cmd.stdErr == null || cmd.stdErr.isEmpty());
+    assertEquals(0, cmd.exitCode, "group list-user returned successfully");
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0].trim().replace("\r", ""));
+    assertEquals("POLICIES", rowHead[1].trim().replace("\r", ""));
+
+    // assert email and policies are correct
+    boolean emailExists = false;
+    for (int i = 1; i < rows.length; i++) {
+      if (rows[i].contains(email)) {
+        emailExists = true;
+        for (var policy : policies) {
+          assertTrue(rows[i].contains(policy.toString()));
```

Comment:
Whenever you use `assertTrue()` or `assertFalse()`, please put a description string saying what you're actually asserting. For example, 
```
assertTrue(rows[i].contains(policy.toString()), "each row contains all policies");
```

Note that in that case you'd probably want to use 

```
assertThat("Row has all policies.", rows[i], hasItems(policy.toString()));
```

Reply:
Hi Jay. thanks for the leading. I will pay attention to the assertion next time. 
due to it could not use hasItems. So I use `assertThat("Row has all policies.", rows[i].contains(policy.toString()));`